tests/commons/j2/test_j2_functions.py

```
@@ -11,290 +12,324 @@
-            self.assertEqual(
-                xsd_fmt(-10, type_name, '"'),
-                '"-10"^^xsd:integer',
-                "bad %s format" % type_name,
-            )
-            self.assertEqual(
-                xsd_fmt(0, type_name, '"'),
-                '"0"^^xsd:integer',
-                "bad %s format" % type_name,
-            )
+def assertFormat(
+    content: any, type_name: str, expected: str, quote: str | None = "'"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)